Pollo/Models/ListDiffableModels/Poll.swift

```diff
@@ -108,19 +88,15 @@ class Poll: Codable {
     func getSelected() -> Any? {
         if userAnswers.isEmpty { return nil }
         guard let googleID = User.currentUser?.id, let answers = userAnswers[googleID], !answers.isEmpty else { return nil }
-        switch type {
-        case .multipleChoice:
-            guard let answer = answers[0].letter else { return nil }
-            return answer
-        }
+        guard let answer = answers[0].letter else { return nil }
```

Comment:
we can probably chain these two `guard let` and i know we check for `!answers.isEmpty` but still might be safer to do `let answer = answers.first?.letter`

Reply:
Yes